example/routing.ts

```diff
@@ -31,7 +31,7 @@ export const routing: Routing = {
     },
   },
   // path /public serves static files from /example/assets
-  public: new ServeStatic(join(__dirname, "assets"), {
+  public: new ServeStatic(join("example", "assets"), {
```

Comment:
that thing does not exist in ESM. The suggested replacement is redundantly complex.

Reply:
not an issue